### PR TITLE
[service.languagepreferencemanager] 1.0.1

### PR DESCRIPTION
**Pull request by rockrider69**
### Description

It provides an easy way to set your preferred audio streams and subtitle languages, valuable for Movies or TV-Shows which includes multiple such tracks.

You can select which audio tracks and subtitles to automatically activate based on your priorities, and define simple conditional rules like "if audio is xxx then activate subtitles yyy" via drop/down lists. Conditions can handle "forced" tracks tag.

More advanced custom rules can be defined as well like fre:non>eng:eng>any:fre>any:eng>.... to handle more than 3 priority levels (See changelog for more on the syntax).

Some new features added recently : reassess the rules on the fly whenever you switch audio track while watching, skip subtitles tracks base on a keyword list vs name content (ex; to skip Signs and Songs tracks if misordered, ...).

Note : this addon has been developed and used for several years. I took over the maintenance few months ago. See README for some credits to previous devs.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.languagepreferencemanager/.gitignore [hunk 1]
```diff
@@ -0,0 +1,39 @@
+*.py[cod]
```
Comment (enen92) [inline service.languagepreferencemanager/.gitignore:1]: please drop from submission

**Review by enen92 — CHANGES_REQUESTED** [review]
New addons are only allowed on the nexus branch now (latest release). Only updates are allowed for matrix

File: service.languagepreferencemanager/.gitignore (hunk 1 above)
Comment (rockrider69) [reply service.languagepreferencemanager/.gitignore:1]: Thanks for the review!
Should I drop the whole .gitignore file or just the 1st line ?

I'll do that before resubmitting the PR to Nexuw according to you other message.

Comment (enen92) [reply service.languagepreferencemanager/.gitignore:1]: whole file

**Comment by rockrider69** [comment]
Just recreated ta new pull request into nexus (and .gitignore file deleted).
Thank you for taking the time to explain, the process is still a bit new to me.

**Pull request closed without merging**